src/main/scala/scalismo/ui/model/TransformationNode.scala

```diff
@@ -64,8 +64,16 @@ class GenericTransformationsNode(override val parent: GroupNode) extends Transfo
     node
   }
 
+  def parentTransformations: PointTransformation = {
+    if (parent != null && parent.parent != null && parent.parent.parent != null && parent.parent.parent.isInstanceOf[GroupNode]) {
```

Comment:
I guess this _code smell_ of parent.parent.parent was not introduced with this PR?